net/net_tcp.c

```diff
@@ -461,10 +462,14 @@ int tcp_conn_get(unsigned int id, struct ip_addr* ip, int port,
 					a, a->parent, a->parent->id, a->port,
 					a->parent->rcv.src_port);
 				print_ip("ip=",&a->parent->rcv.src_ip,"\n");
+				if (send_sock != NULL && a->parent->send_sock != NULL)
+					print_ip("requered send_sock ip=",&send_sock->address,"\n");
```

Comment:
Assuming that should be `required`?

Reply:
This has been fixed before merging, but replaced with `requested` instead of `required`.